### ISSUE TITLE
WCAG 4.1.2: Ensure buttons have discernible text (.-left-10.h-7.w-7)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#89]
#### Issue
`Ensure buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app/guide/works)
#### Element path
tr[data-index="0"] > .p-2\.5.md\:p-3:nth-child(1) > .w-full.md\:text-sm.text-xs > .group\/cell.min-h-\[40px\].h-full > .-left-10.h-7.w-7
#### Snippet
`<button data-slot="tooltip-trigger" data-variant="ghost" data-size="icon" class="items-center justify-center gap-2 whitespace-nowrap text-sm font-medium disabled:pointer-events-none disabled:opacity-50 [&amp;_svg]:pointer-events-none [&amp;_svg:not([cla...`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element does not have an implicit (wrapped) <label>
      Element does not have an explicit
